browser/new-command.js

```diff
@@ -107,7 +110,7 @@ class ThingTalkTrainer {
                 results.append($('<li>').append(link));
             }
```

Comment:
not in this line, but probably catch errors such as "Failed to reconstruct canonical" and give users a hint?

Reply:
Barring bugs, "Failed to reconstruct canonical" should happen only if the command parses to an unapproved device and you're not logged in. I'm not sure how to handle that case though, cause it could happen down the beam, or it could just be a bad parse.